microsoftgraph/docs-conceptual/tutorial-grant-app-only-api-permissions.md

```diff
@@ -0,0 +1,145 @@
+
+In this tutorial, you'll grant app roles that are exposed by an API to an app. App roles, also called application permissions, or direct access permissions, allow an app to call an API with its own identity.
+
+## Prerequisites
+
+To successfully complete this tutorial, make sure you have the required prerequisites:
+
+1. A working Azure AD tenant.
+1. Microsoft Graph PowerShell SDK is installed. Follow the [Install the Microsoft Graph PowerShell SDK](installation.md) guide to install the SDK.
+1. Microsoft Graph PowerShell using a global administrator role and the appropriate permissions. For this tutorial, the `Application.Read.All` and `AppRoleAssignment.ReadWrite.All` delegated permissions are required. To set the permissions in Microsoft Graph PowerShell, run;
+
+    ```powershell
+    Connect-MgGraph -Scopes "Application.ReadWrite.All", "AppRoleAssignment.ReadWrite.All"
+    ```
+
+    Select **Consent on behalf of your organization** before accepting in the login dialog box.
+
+>[!Caution]
+>The `AppRoleAssignment.ReadWrite.All` permission allows an app or a service to manage permission grants and elevate privileges for any app, user, or group in your organization. Only appropriate users should access apps that have been granted this permission.
+
+## Step 1: Get the app roles of the resource service principal
+
+Before you can grant app roles, you must first identify the app roles to grant and the resource service principal that exposes the app roles. App roles are defined in the `appRoles` object of a service principal. 
+
+In this article, you'll use the `Microsoft Graph` service principal in the tenant as your resource service principal.
+
+```powershell
+Get-MgServicePrincipal -Filter "displayName eq 'Microsoft Graph'" -Property AppRoles | Select -ExpandProperty appRoles |fl
+```
+
+```Output
+AllowedMemberTypes   : {Application}
+Description          : Allows the app to read and update user profiles without a signed in user.
+DisplayName          : Read and write all users' full profiles
+Id                   : 741f803b-c850-494e-b5df-cde7c675a1ca
+IsEnabled            : True
+Origin               : Application
+Value                : User.ReadWrite.All
+AdditionalProperties : {}
+
+AllowedMemberTypes   : {Application}
+Description          : Allows the app to read user profiles without a signed in user.
+DisplayName          : Read all users' full profiles
+Id                   : df021288-bdef-4463-88db-98f22de89214
+IsEnabled            : True
+Origin               : Application
+Value                : User.Read.All
+AdditionalProperties : {}
+```
+
+## Step 2: Create a client service principal
+
+The first step in granting consent is to [create the service principal](/powershell/module/microsoft.graph.applications/new-mgserviceprincipal?view=graph-powershell-1.0&preserve-view=true). To do so, you'll need the `App Id` of your application.
```

Comment:
I would slightly expand this - "[create a service principal] for the app that you'll grant permissions."